nephio-webui/Kptfile

```diff
@@ -7,4 +7,9 @@ metadata:
 info:
   site: https://nephio.org
   description: Package for the Nephio Web UI.
-pipeline: {}
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:v0.5.0
```

Comment:
I don't know how it works, but this requires docker...

Reply:
As opposed to podman? Can you try setting KPT_FN_RUNTIME=podman ?

Reply:
https://kpt.dev/reference/cli/fn/render/?id=environment-variables

Reply:
`KPT_FN_RUNTIME=podman kpt fn render .` did the trick; thank you. 
We should probably document somewhere how to customize fn runtime.

Reply:
I will add it to the prerequisites for the byoc flow